src/main/java/lotto/domain/Money.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain;
+
+import lotto.enums.ErrorMessage;
+
+public class Money {
+    private final int cash;
+    private final int tryCount;
+
+    public Money(int cash, int tryCount) {
+        validateIsDivisible(cash);
+        this.cash = cash;
+        this.tryCount = cash / 1000;
+    }
+
+    public int getTryCount() {
+        return tryCount;
+    }
+
+    public int getCash() {
+        return cash;
+    }
+
+    private void validateIsDivisible(int cash) {
+        if (cash % 1000 == 0) {
+            return;
+        }
+
+        throw new IllegalArgumentException(ErrorMessage.INVALID_NONE_DIVISIBLE.getMessage() + cash);
+    }
```

Comment:
이 부분은 이렇게 처리해도 좋을 듯 싶네요!
```suggestion
    private void validateIsDivisible(int cash) {
        if (cash % 1000 != 0) {
            throw new IllegalArgumentException(ErrorMessage.INVALID_NONE_DIVISIBLE.getMessage() + cash);
        }
    }
```

Reply:
더 깔끔해졌네요! 감사합니다